<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Step4Widget</class>
 <widget class="QWidget" name="Step4Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>901</width>
    <height>563</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Step4Widget</string>
  </property>
  <widget class="QPushButton" name="buttonExport">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>450</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Export</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonPlay">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>370</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&gt;</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>60</y>
     <width>651</width>
     <height>331</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="Step4RenderArea" name="imageWidget" native="true"/>
    </item>
    <item>
     <widget class="QSlider" name="imageSlider">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="buttonRender">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>20</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Render</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>Step4RenderArea</class>
   <extends>QWidget</extends>
   <header>Step4Widget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
